src/app/ui/components/list/styles.ts

```diff
@@ -0,0 +1,59 @@
+import { css } from 'leather-styles/css';
```

Comment:
can we name file `list.styles.ts` or something, otherwise we end up with hundeds of `styles.ts` which makes search harder